logdoctor/translations/LogDoctor_ru_RU.ts

```diff
@@ -33,2145 +29,2004 @@
     </message>
     <message>
         <source>BSD</source>
-        <translation type="unfinished">BSD</translation>
+        <translation>BSD</translation>
     </message>
     <message>
         <source>Windows</source>
-        <translation type="unfinished">Windows</translation>
+        <translation>Windows</translation>
     </message>
     <message>
         <source>OSX</source>
-        <translation type="unfinished">OSX</translation>
+        <translation>OSX</translation>
     </message>
     <message>
         <source>Statistics:</source>
-        <translation type="unfinished">Статистика:</translation>
+        <translation>Статистика:</translation>
     </message>
     <message>
         <source>Warnings</source>
-        <translation type="unfinished">Предупреждения</translation>
+        <translation>Предупреждения</translation>
     </message>
     <message>
         <source>Speed</source>
-        <translation type="unfinished">Скорость</translation>
+        <translation>Скорость</translation>
     </message>
     <message>
         <source>Counts</source>
-        <translation type="unfinished">Графиня</translation>
+        <translation>Подсчёты</translation>
     </message>
     <message>
         <source>Daytime</source>
-        <translation type="unfinished">дневное время</translation>
+        <translation>Время суток</translation>
     </message>
     <message>
         <source>Relational</source>
-        <translation type="unfinished">относительный</translation>
+        <translation>Связи</translation>
     </message>
     <message>
         <source>Globals</source>
-        <translation type="unfinished">Глобал</translation>
+        <translation>Глобальные переменные</translation>
     </message>
     <message>
         <source>Translations:</source>
-        <translation type="unfinished">Переводы:</translation>
+        <translation>Переводы:</translation>
     </message>
     <message>
         <source>Italian</source>
-        <translation type="unfinished">итальянский</translation>
+        <translation>Итальянский</translation>
     </message>
     <message>
         <source>Spanish</source>
-        <translation type="unfinished">испанский</translation>
+        <translation>Испанский</translation>
     </message>
     <message>
         <source>French</source>
-        <translation type="unfinished">французский</translation>
+        <translation>Французский</translation>
     </message>
     <message>
         <source>Tools:</source>
-        <translation type="unfinished">Инструменты:</translation>
+        <translation>Инструменты:</translation>
     </message>
     <message>
         <source>Block note</source>
-        <translation type="unfinished">Блокнот</translation>
+        <translation>Блокнот</translation>
     </message>
     <message>
         <source>Utilities:</source>
-        <translation type="unfinished">Коммунальные услуги:</translation>
+        <translation>Утилиты</translation>
```

Comment:
```suggestion
        <translation>Утилиты:</translation>
```
final colon is missing